addons/web/static/src/views/fields/file_handler.js

```diff
@@ -61,6 +61,8 @@ export class FileUploader extends Component {
                     data: data.split(",")[1],
                     objectUrl: file.type === "application/pdf" ? URL.createObjectURL(file) : null,
                 });
+            } catch {
```

Comment:
do we want to silently catch errors when uploading?

Reply:
The assert it no ?

Reply:
no hablo espanol